package.json

```
@@ -2,20 +2,31 @@
   "name": "deploy-action",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "lib/index.js",
   "scripts": {
-    "build": "ncc build index.js",
-    "prettier:fix": "prettier --write index.js"
+    "build": "tsc && ncc build",
```

```suggestion
    "build": "tsc && ncc build",
    "build:prod": "ncc build src/index.ts --minify",
```
suggestion: WDYT for reducing the size of the artifact seeing as we have to commit it?

---

I don't really see benefits in minifying. 
* Having less readable but shorter commits. Not a benefit in my opinion.
* Theoretically there could be some benefits in starting up because I would imagine the code still needs to be loaded to the runner but I would imagine this is not perceivable. 
* Minfying might make debugging more difficult in some cases... 

I'm resolving this without making the change but you can reopen this if you feel that I closed this too quickly.

---

I'm alright with us not minifying, though I'll note some responses here while it's on-topic

> Having less readable but shorter commits. Not a benefit in my opinion.

I don't expect us to read the collated (and potentially minified) code. In other words, I believe `dist/index.js` is hidden from us by GitHub without that detracting from our reviews.

> Theoretically there could be some benefits in starting up because I would imagine the code still needs to be loaded to the runner but I would imagine this is not perceivable.

Agreed, I don't think that'll be perceivable from the runner perspective.

> Minfying might make debugging more difficult in some cases...

I'm not convinced it won't be already, but it is a valid point that it'll make it harder to read error messages.